carmine-api-airdrop/src/merkle_tree.rs

```diff
@@ -0,0 +1,160 @@
+use serde::Deserialize;
+use starknet_crypto::{pedersen_hash, FieldElement};
+use std::fs::File;
+
+#[derive(Deserialize, Debug, Clone)]
+pub struct Airdrop {
+    pub address: String,
+    amount: u32,
```

Comment:
Currently, both "address" and "amount" is a string, I assumed it would be stored in a JSON file, where it makes sense to be stored as such.
Let's talk about this some more.